src/Internal/Payment/Change.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Class Change
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal\Payment;
+
+/**
+ * A change object used for logging.
+ * Key, former and current values are set as string.
+ */
+class Change {
+
+	/**
+	 * Identifier or key for the changed value.
+	 *
+	 * @var string
+	 */
+	private $key;
+
+	/**
+	 * Former value for the key.
+	 *
+	 * @var string|null
+	 */
+	private $old_value;
+
+	/**
+	 * Current value, updated aganist the key.
+	 *
+	 * @var string|null
+	 */
+	private $new_value;
+
+	/**
+	 * Constructs the class, with key, former and current values.
+	 *
+	 * @param string      $key Identifier or key for the changed value.
+	 * @param string|null $old_value Former value for the key.
+	 * @param string|null $new_value Current value, updated aganist the key.
```

Comment:
It seems $old_value and $new_value can be any type, not just string. Looking at `PaymentContext` now, it can be any scalar value as well as array and object (such as `WC_Payments_API_Abstract_Intention`).

Reply:
No longer using string